app/models/group_offer.rb

```diff
@@ -83,4 +88,15 @@ def volunteers_with_roles
       end
     end.compact.join(', ')
   end
+
+  private
+
+  def department_manager_has_department?
+    if user.department.blank?
+      errors.add(:user_no_department, "#{I18n.t('role.department_manager')} müssen einem Standort zugeteilt sein, "\
+        "bevor sie #{I18n.t('group_offers', count: 2)} erfassen können.")
+    elsif !user.department.include?(department)
+      errors.add(:user_wrong_department, 'Nicht der richtige Standort.')
```

Comment:
Do we need translations for the errors? You can use [string interpolation](http://www.rubydoc.info/gems/i18n/I18n/Base#translate-instance_method) for the dynamic parts.

Reply:
Also you should probably do `errors.add(:department, ...`, doesn't make much of a difference but the first argument is supposed to be the attribute with the error.